src/cli/cmds/synth.ts

```diff
@@ -18,7 +23,16 @@ class Command implements yargs.CommandModule {
     .option('stdout', { type: 'boolean', required: false, desc: 'Write synthesized manifests to STDOUT instead of the output directory', alias: 'p' })
     .option('plugins-dir', { default: config.pluginsDirectory, required: false, desc: 'Directory to store cdk8s plugins.' })
     .option('validate', { type: 'boolean', default: true, required: false, desc: 'Apply validation plugins on the resulting manifests (use --no-validate to disable)' })
-    .option('validation-reports-output-file', { required: false, desc: 'File to write a JSON representation of the validation reports to' });
+    .option('validation-reports-output-file', { required: false, desc: 'File to write a JSON representation of the validation reports to' })
+    .option('format', {
+      default: config.format,
+      required: false,
+      desc: 'Synthesis format for Kubernetes manifests.',
+      choices: ['cdk8s', 'helm'],
```

Comment:
```suggestion
      choices: ['plain', 'helm'],
```

`cdk8s` is not a format, we produce "plain" manifests by default.